src/main.ts

```diff
@@ -237,7 +237,7 @@ export class CustomSonosCard extends LitElement {
         max-width: 25%;
       }
       .title {
-        margin-top: 10px;
+        margin-top: 10px 0;
```

Comment:
What does the extra zero do?

Reply:
I see that I've made a mistake, it needs to be `margin: 10px 0;` so in theory that makes it `margin: 10px 0 10px 0;`